settings.gradle

```
@@ -1,2 +1,3 @@
+rootProject.name = 'nextCloud'
```

```suggestion
rootProject.name = 'Nextcloud'
```

...so it fits the official naming schema 👍

---

Now it is